#!/usr/bin/env python

PACKAGE = "first_project"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# Define the parameters for dynamic reconfiguration
gen.add("lidar_frame", str_t, 0, "Frame to which the lidar is connected", "/wheel_odom", edit_method="combo")

# Generate the dynamic reconfigure file
gen.generate(PACKAGE, "LidarVisualization", "LidarVisualizationConfig")
